core/pom.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+  	<groupId>pentaho</groupId>
+  	<artifactId>pentaho-pdi-platform-plugin</artifactId>
+  	<version>8.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>pentaho</groupId>
+  <artifactId>DIS</artifactId>
+  <version>8.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>DIS</name>
+
+  <dependencies>
+    <!-- Kettle dependencies -->
+	<dependency>
+	  <groupId>pentaho-kettle</groupId>
+	  <artifactId>kettle-engine</artifactId>
+	  <version>${project.version}</version>
+	</dependency>
+	<dependency>
+	  <groupId>pentaho-kettle</groupId>
+	  <artifactId>kettle-core</artifactId>
+	  <version>${project.version}</version>
+	</dependency>
+	<dependency>
+	  <groupId>org.pentaho.di.plugins</groupId>
+	  <artifactId>kettle-log4j-core</artifactId>
+	  <version>${project.version}</version>
+	</dependency>	
+    <!-- Pentaho dependencies -->
+	<dependency>
+	  <groupId>pentaho</groupId>
+	  <artifactId>pentaho-platform-api</artifactId>
+	  <version>${project.version}</version>
+	</dependency>
+	<dependency>
+	  <groupId>pentaho</groupId>
+	  <artifactId>pentaho-platform-core</artifactId>
+	  <version>${project.version}</version>
+	</dependency>
+	<dependency>
+	  <groupId>pentaho</groupId>
+	  <artifactId>pentaho-platform-repository</artifactId>
+	  <version>${project.version}</version>
+	</dependency>
+	<dependency>
+	  <groupId>pentaho</groupId>
+	  <artifactId>pentaho-platform-extensions</artifactId>
+	  <version>${project.version}</version>
+	</dependency>
+    <!-- Third-party dependencies -->
+	<dependency>
+	  <groupId>org.apache.commons</groupId>
+	  <artifactId>commons-collections4</artifactId>
+	  <version>${commons.version}</version>
+	</dependency>
+    <!-- Test dependencies -->
+	<dependency>
+	  <groupId>pentaho</groupId>
+	  <artifactId>pentaho-platform-core</artifactId>
+	  <version>${project.version}</version>
+	  <classifier>tests</classifier>
+	</dependency>
+	<dependency>
+	  <groupId>junit</groupId>
+	  <artifactId>junit</artifactId>
+	  <version>${junit.version}</version>
+	  <scope>test</scope>
+	</dependency>
+	<dependency>
+	  <groupId>org.hsqldb</groupId>
```

Comment:
Not sure I need to exclude anything here the artifact is correct and isn't bringing in additional dependencies.